packages/react/src/primitives/types/autocomplete.ts

```diff
@@ -144,3 +144,26 @@ export interface UseAutocompleteProps extends Partial<AutocompleteProps> {
 
   onClick?: React.MouseEventHandler<HTMLInputElement>;
 }
+
+type SetStateAction<T> = React.Dispatch<React.SetStateAction<T>>;
+
+export interface UseAutocomplete {
```

Comment:
Maybe we should keep this stay in the same file as `useAutocomplete.tsx`?

Reply:
I placed it with the rest of the `AutoComplete` component types for consistency. Don't feel strongly but is there a reason not include with the rest of the types?

Reply:
No reason but looking at the `useCheckbox`, we group its type together

Reply:
That's valid. I'm more apt to move `UseCheckbox` to the `Checkbox` types though